keep/providers/netdata_provider/netdata_provider.py

```diff
@@ -0,0 +1,91 @@
+"""
```

Comment:
1. add some alerts example (see `alerts_mock.py`) so that I can try it
2. can you add a README how to spin up netdata and try it out? see ntfy provider